server/apps/call.go

```diff
@@ -3,15 +3,19 @@ package apps
 import "github.com/mattermost/mattermost-plugin-apps/server/api"
 
 func (s *service) Call(call *api.Call) (*api.CallResponse, error) {
-	var err error
+	// TODO <><> get a cached function
```

Comment:
Really, Expand will likely end up served with the form definition, back into `Call`?